doc/maintenance.md

```diff
@@ -0,0 +1,539 @@
+- MUST include in the request body:
+  - For `PUT`, **one** Item Scheme in the SDMX format indicated in the `Content-type` header and of the Item Scheme type indicated in the resource, e.g. a Concept Scheme under resource `/conceptscheme`;
+  - For `POST`, one or more Item Schemes in the SDMX format indicated in the `Content-type` header and
+    - of the Item Scheme type indicated in the resource, in case of a specific Item Scheme resource, e.g. a Codelist under resource `/structure/codelist/`
+    - of any Item Scheme type, in case of the `/structure` resource.
+
+##### Server
+
+In response to an Item Scheme partial update, the server:
+
+- MUST respond with `200` in case of successful update;
+- MUST return a `SubmitStructureResponse` message with the result of the action, according to the `Accept` header, or the default, if the `Accept` type is not supported (currently available only in SDMX-ML);
+- MUST set the `Content-type` according to the returned format;
+- MUST respond with `422` in case of resource type mismatch, i.e. the artefact identified in the URL does not match either to the resource type or identification of the included SDMX Artefact.
+- MUST respond with `404` if the resource was not found;
+
+##### Response
+
+The `SubmitStructureResponse` message must be returned in any case (success, partial success, failure).
+This is defined as part of the `RegistryInterface` messages.
+The details of the message are explained in section **Structure response message**, below.
+
+The following matrix summarises the returned `HTTP` response codes.
+
+| Method | Exists | Versioning respected | Is Referenced | Refs exist/ provided | Response Code |
+|---|---|---|---|---|---|
+| PUT/POST | T | T | F | T | `200` (successful) or `207` (partially successful)  |
+| PUT/POST | T | T | F | F | `409` |
+| PUT/POST | T | T | T | T | `200` if update is possible - `409` if references would break |
+| PUT/POST | T | F | - | - | `409` in case of changes that violate the versioning requirements |
+| PUT/POST | - | - | - | - | `422` see section **Server** |
+| PUT | F | - | - | - | `404` |
+
+<sup>T: True, F: False, I: Irrelevant, -: Not applicable</sup>
+
+
+### DELETE
+
+Always concerns one Maintainable Artefact or one Item. 
+For example:
+- A fully identified Maintainable Artefact, e.g. `/structure/codelist/SDMX/CL_FREQ/1.0`
+- A fully identified Item, e.g. `/structure/codelist/SDMX/CL_FREQ/1.0/M`
+
+In case the deleted Item was acting as a parent to other Item(s), then the server should make sure that:
+- in case of flat Item Schemes the children become orphans, i.e. their parent is removed;
+- in case of nested Item Schemes, all children are also deleted.
+
+##### Client
+
+In order to delete an Artefact, the client:
+
+- MAY set the `Accept` header to indicate the preferred response format;
+- MUST fully identify exactly **one** Maintainable Artefact or **one** Item, by means of the proper URL;
```

Comment:
Why not support deleting several items in one go, e.g. with /OECD/CL/1.0.0/A,B,C?